src/Xamarin.Android.Build.Tasks/Xamarin.Android.Sdk.props

```diff
@@ -0,0 +1,58 @@
+<!--
+***********************************************************************************************
+Xamarin.Android.Sdk.props
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+  created a backup copy.  Incorrect changes to this file will make it
+  impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (C) 2011-2016 Xamarin. All rights reserved.
+***********************************************************************************************
+-->
+<Project InitialTargets="RedirectMonoAndroidSdkPaths;RemoveSdksCache" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+	<PropertyGroup>
```

Comment:
This whole file was intended only to work on VS2017+, which is true as long as `'$(VsInstallRoot)' != ''`. Maybe the property groups and targets in this file should have the condition, just to make it more explicit and avoid unintentional usage outside of that scenario?